src/AppBundle/Form/MembershipRequestType.php

```diff
@@ -23,7 +23,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('gender', GenderType::class)
             ->add('firstName', TextType::class)
             ->add('lastName', TextType::class)
-            ->add('emailAddress', EmailType::class)
+            ->add('emailAddress', EmailType::class, [
+                'empty_data' => '',
```

Comment:
Bug fix, it activates the transformer `null => ''`, see https://github.com/symfony/symfony/pull/18357 :)